pytest.ini

```diff
@@ -9,3 +9,6 @@ filterwarnings =
 addopts = -v
 markers =
     skip_clickhouse_client: marks tests as requiring clickhouse client to be skipped
+    weave_client: alias for `client` fixture used for CI dispatching. Do not use directly in code.
+    flaky: marks tests as flaky (CI will retry them if they fail)
```

Comment:
I think you get this for free with:
https://pypi.org/project/pytest-rerunfailures/

We should add it back to `requirements.test.txt`

Reply:
hmmm, we have `pytest.mark.flaky` in a few places... does you comment imply these are not actually working right now?

Reply:
Yeah if the plugin is not present then the marks are ignored (tests are still run, but no retries for flake)

You can confirm by doing:
```
tox -e trace-py312 -- --markers
```

The flaky marker does not show up which means it's being ignored atm

Reply:
ok, i can add this to this PR and remove this line

Reply:
done